spark3/src/main/java/org/apache/iceberg/actions/Spark3MigrateAction.java

```diff
@@ -46,11 +46,11 @@
  * previously referred to a non-iceberg table will refer to the newly migrated iceberg
  * table.
  */
-class Spark3MigrateAction extends Spark3CreateAction {
+public class Spark3MigrateAction extends Spark3CreateAction {
```

Comment:
Hm. I would prefer not to make these public, but I see that this needs the identifier that has already been parsed to do the custom catalog validation. Should be fine for now, but we should keep this in mind for when we fix the `Actions` API.

Reply:
I think we should really have all of these spark implementations of actions in a "spark" package so we don't have to public everything.

Reply:
This is another point to fix in actions before we release 0.11.